chalice/awsclient.py

```diff
@@ -100,6 +101,14 @@ def get_function_configuration(self, name):
             FunctionName=name)
         return response
 
+    def create_vpc_config(self, security_group_ids, subnet_ids):
+        vpc_config = {}
+        if subnet_ids is not None:
+            vpc_config['SubnetIds'] = subnet_ids
+        if security_group_ids is not None:
```

Comment:
As per [AWS documentation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-vpcconfig.html), both SubnetIds and SecurityGroupIds are mandatory parameters for VPC configuration.  IMO We should validate if both are present.